InternetHospital.BusinessLogic/Services/UploadingService.cs

```diff
@@ -52,10 +51,9 @@ public async Task<User> UploadAvatar(IFormFile image, User user)
             {
                 Directory.CreateDirectory(fileDestDir);
             }
-
-            int fileNameLength = user.UserName.Length < IMAGE_MAX_LENGTH ? user.UserName.Length : IMAGE_MAX_LENGTH;
```

Comment:
remove fileNameLength value in this method

Reply:
I will delete it while resolving conflicts